Src/Eiffel/eiffel/AST/visitor/ast_feature_checker_generator.e

```diff
@@ -1597,6 +1597,13 @@ feature {NONE} -- Implementation
 									has_vucr := w /= error_handler.warning_level
 								end
 							end
+						elseif
+							l_found_feature.is_once and then attached l_found_feature.written_class.creators as l_found_creators and then l_found_creators.has (l_found_feature.feature_name) and then
+							attached context.current_class.creators as l_creators and then not l_creators.has (current_feature.feature_name) then
```

Comment:
It's unclear why we look into creation procedures of the current class.

Reply:
We have a rule that said:  Once creation procedures must be immediate (i.e. not inherited).

Reply:
Right, but this should be checked somewhere else, no?